libs/sdk-ui-ext/src/internal/interfaces/Visualization.ts

```diff
@@ -332,6 +335,8 @@ export interface IVisualization {
         referencePoint: IReferencePoint,
         previousReferencePoint?: IReferencePoint,
     ): Promise<IExtendedReferencePoint>;
+
+    getInsightWithDrillDownApplied(source: IInsight, drillDownContext: IDrillDownContext): IInsight;
```

Comment:
need contractual docs for this & description what's the purpose and what is the expected behavior of the impl